docs/pages/zh-CN/ui/asciinema-player/index.md

```diff
@@ -46,39 +46,126 @@ yarn add @nolebase/ui-asciinema
 
 ## 用法
 
-```js
+Add the package to Vite's `ssr.noExternal` configuration. Without this, your site may not build.
+
+```ts
 // vite.config.ts
 export default defineConfig({
-  ssr: {
-    noExternal: [
-      // If there are other packages that need to be processed by Vite, you can add them here.
-      '@nolebase/ui-asciinema',
-    ],
+  vite: {
+    ssr: {
+      noExternal: [
+        "@nolebase/ui-asciinema",
+      ],
+    },
   },
 })
 ```
 
-```vue
-<script setup>
-// somewhere.vue
-import { NuAsciinemaPlayer } from '@nolebase/ui-asciinema'
-import 'asciinema-player/dist/bundle/asciinema-player.css'
-</script>
-
-<template>
-  <NuAsciinemaPlayer
-    src="/asciinema/test-nyancat.cast"
-    :preload="true"
-    :cols="400"
-    :rows="40"
-    :auto-play="true"
-    :controls="true"
-    :terminal-font-size="'12px'"
-    :loop="true"
-  />
-</template>
+Refs:
+
+- [VitePress's Vite configuration docs](https://vitepress.dev/reference/site-config#vite)
+- [Vite's `ssr.noExternal` config docs](https://vitejs.dev/guide/ssr.html#ssr-externals)
+
+If a component is only used by a few pages, it's recommended to explicitly import them where they are used. This allows them to be properly code-split and only loaded when the relevant pages are shown:
```

Comment:
This English text comes from https://vitepress.dev/guide/using-vue#importing-in-markdown